### PR TITLE
Migrate some of the edited pages

### PR DESCRIPTION
**Pull request by d-laing**
# Description

Migrating some of the pages from the old site.

## Type of change

- [x] New document(s)/updating existing
- [x] Fixes

### CODE REVIEW

File: docs/integrations/commerce/shopify/commerce-shopify-setup.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 createdAt: "2021-01-14T16:41:39.945Z"
 updatedAt: "2023-01-06T16:44:14.501Z"
 ---
 
 Set up the Shopify integration to retrieve commerce data from customers who sell their products on Shopify's ecommerce platform.
 
-:::caution
+This article explains how to set up the Shopify integration using the _public apps_ connection method.
 
+:::caution
```
Comment (pzaichkina) [inline docs/integrations/commerce/shopify/commerce-shopify-setup.md:12]:
```suggestion
:::caution Before you begin
```

same as above

File: docs/integrations/commerce/shopify/commerce-shopify-setup.md [hunk 2]
```diff
@@ -20,13 +21,15 @@ To set up the Shopify integration, perform the following tasks in Shopify and th
...
 
-## Create a Shopify app
+## Create a Shopify public app
 
 Create and configure an app in Shopify. When the app is created you can view the secure app credentials.
 
 1. Log in to the [Shopify Partners](https://developers.shopify.com/) site using your Shopify partner account credentials.
 
-   > [Create a partner account](https://partners.shopify.com/signup/developer) if you don't already have one.
+   :::note
```
Comment (pzaichkina) [inline docs/integrations/commerce/shopify/commerce-shopify-setup.md:30]:
```suggestion
   :::note Shopify partner account
```

File: docs/integrations/commerce/shopify/commerce-shopify.md [hunk 3]
```diff
@@ -5,27 +5,66 @@ createdAt: "2021-01-13T16:44:23.830Z"
...
+View the coverage of our Shopify integration in the <a class="external" href="https://knowledge.codat.io/supported-features/commerce?view=tab-by-integration&integrationKey=fztf" target="_blank">Data Coverage Explorer</a>.
 
-## Set up the integration
+## Connection methods
 
-See [Set up the Shopify integration](/integrations/commerce/shopify/commerce-shopify-setup) to learn how to set up and enable the integration.
+Our Shopify integration allows your merchants to connect their commerce data using one of two methods:
 
-## Underlying provider requirements
+- [Custom apps](commerce-shopify-custom-apps)
```
Comment (pzaichkina) [inline docs/integrations/commerce/shopify/commerce-shopify.md:20]:
```suggestion
- [Custom apps](/integrations/commerce/shopify/commerce-shopify-custom-apps)
```

File: docs/integrations/commerce/shopify/commerce-shopify.md [hunk 4]
```diff
@@ -5,27 +5,66 @@ createdAt: "2021-01-13T16:44:23.830Z"
...
+- [Public apps](commerce-shopify-setup)
```
Comment (pzaichkina) [inline docs/integrations/commerce/shopify/commerce-shopify.md:21]:
```suggestion
- [Public apps](/integrations/commerce/shopify/commerce-shopify-setup)
```

File: docs/integrations/commerce/shopify/commerce-shopify.md [hunk 5]
```diff
@@ -5,27 +5,66 @@ createdAt: "2021-01-13T16:44:23.830Z"
...
+
+There are no additional provider requirements when using this connection method.
+
+To connect their commerce data using a custom app, the merchant does the following:
+
+1. Creates a custom app and assigns it the required API scopes.
+2. Gets the API access token for the app.
+3. Enters the access token in Link.
+
+To get started, see [Set up Shopify using custom apps](commerce-shopify-custom-apps).
```
Comment (pzaichkina) [inline docs/integrations/commerce/shopify/commerce-shopify.md:37]:
```suggestion
To get started, see [Set up Shopify using custom apps](/integrations/commerce/shopify/commerce-shopify-custom-apps).
```

File: docs/integrations/commerce/shopify/commerce-shopify.md [hunk 6]
```diff
@@ -5,27 +5,66 @@ createdAt: "2021-01-13T16:44:23.830Z"
...
+Additionally, if you need to access over 60 days' of orders, you must submit a _Read all orders_ request as noted in [Create a Shopify public app](commerce-shopify-setup#create-a-shopify-public-app). You'll receive an email from Shopify when your request has been processed; this might take up to seven business days.
+
+:::caution Important
+
+Before setting up the integration, review the preceding requirements to make sure your use case is in accordance with Shopify's Terms of Service. **Codat can't guarantee that Shopify will approve your public app or grant access to over 60 days' of order data.**
+
+:::
 
-Before setting up the integration, review the preceding requirements to make sure your use case is in accordance with Shopify's Terms of Service. Codat can't guarantee that Shopify will approve your app or grant access to all order data.
+To get started, see [Set up Shopify using public apps](commerce-shopify-setup).
```
Comment (pzaichkina) [inline docs/integrations/commerce/shopify/commerce-shopify.md:70]:
```suggestion
To get started, see [Set up Shopify using public apps](/integrations/commerce/shopify/commerce-shopify-setup).
```

File: docs/other/user-management/sso.md [hunk 7]
```diff
@@ -7,41 +7,49 @@ updatedAt: "2022-12-19T06:15:27.799Z"
...
+
+To manage third-party SSO services, perform the following steps in the Codat Portal:
 
-Both Google and Microsoft sign-in services are enabled by default in the Codat Portal. We recommend keeping them enabled to make the most of the benefits they provide. Administrators can also turn off password-based sign-in completely and force all their users to sign in with their Google or Microsoft account.
+1. Go to **Settings > Organization settings > Users**.
+2. Click **Manage sign-in** (only available for Administrators).
+3. In the **Manage sign-in** dialog, use the toggles to switch the services on and off.
 
-As long as the email addresses are the same, attempting to sign in with a Google or Microsoft account will automatically link it to the existing Codat account.
+:::note
```
Comment (pzaichkina) [inline docs/other/user-management/sso.md:43]:
```suggestion
:::note Custom login message
```

File: docs/sfc/build-with-sync-for-commerce/customizing-the-sync-configuration-flow.md [hunk 8]
```diff
@@ -5,11 +5,12 @@ createdAt: "2022-11-16T20:24:20.979Z"
 updatedAt: "2023-01-16T17:31:55.362Z"
 ---
 
-:::caution Customization vs. features
+:::caution
```
Comment (pzaichkina) [inline docs/sfc/build-with-sync-for-commerce/customizing-the-sync-configuration-flow.md:8]:
```suggestion
:::caution Customization vs. features
```

**Review by pzaichkina — COMMENTED** [review]
Made some suggestions

File: docs/integrations/accounting/quickbooksdesktop/install-qbd-connector-right-networks.md [hunk 9]
```diff
@@ -73,7 +75,13 @@ If you're using Link, you can enable the Right Networks installation path in the
 1. Sign in to the <a  class="external" href="https://app.codat.io" target="_blank">Codat Portal</a>.
 2. In the navigation bar, select **Settings > Integrations > Accounting**.
 3. Click **Manage** next to the **QuickBooks Desktop** integration.
-4. Select **Enable your customer users to see the Right Networks install guide in the QBD connector download UI**.
+4. Select the checkbox named **Enable your customer users to see the Right Networks install guide in the QBD connector download UI**.
+
+   :::note
```
[the remark this answers is not in the document]
Comment (d-laing) [reply docs/integrations/accounting/quickbooksdesktop/install-qbd-connector-right-networks.md:80]: OK, I'll watch out for these in future.

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**